app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -665,6 +670,10 @@ class ProjectActivity :
         startActivityWithTransition(intent, R.anim.slide_in_right, R.anim.fade_out_slide_out_left)
     }
 
+    private fun startCommentsThreadedActivity(projectAndData: Pair<Project, ProjectData>) {
+        // TODO: Start the new activity defined in https://kickstarter.atlassian.net/browse/NT-1920
```

Comment:
@leighdouglas you can start the new activity in this method, for the task https://kickstarter.atlassian.net/browse/NT-1920

Reply:
Got it! will do as soon as this lands and will update my PR